mailu/README.md

```diff
@@ -76,3 +79,13 @@ If neither `persistence.hostPath` nor `persistence.existingClaim` is set, a new
 can be overridden with `persistence.claimNameOverride`.
 
 The `persistence.storageClass` is not set by default. It can be set to `-` to have an empty storageClassName or to anything else to use this name.
+
+## Ingress
+
+The default ingress is handled externally. In some situations, this is problematic, such as when webmail should be accessible
+ on the same address as the exposed ports. Kubernetes services cannot provide such capabilities without vendor-specific annotations.
+ 
+By setting `ingress.externalIngress` to false, the internal NGINX instance provided by `front` will configure TLS according to
+ `ingress.tlsFlavor` and redirect `http` scheme connections to `https`. 
+ 
+ CAUTION: This configuration exposes `/admin` to all clients with access to the web UI.
```

Comment:
Could you elaborate what this exactly means? Is this a security issue? Or is it the same as with the current ingress behaviour?

Reply:
It’s the same as the current behavior, which also doesn’t document the fact that admin is exposed. I’m happy to document it differently if it is confusing or alarming. Just thought users ought to know that they are exposing it.